doc/articles/guides/community-tutorials.md

```diff
@@ -1,9 +1,9 @@
 ## Community Tutorials
 
-This page lists links our community provided tutorials.
+This page lists links to our community provided tutorials.
 
 > [!NOTE]
-> Please use this page to submit links to your blogs, tutorials and other Uno Platform advocacy as part of Hacktoberfest 2022.
+> Please use this page to submit links to your blogs, articles, and other resources as contribution to Uno Platform advocacy for Hacktoberfest 2022.
```

Comment:
Should we remove this note now that Hacktoberfest 2022 already ended?

Reply:
I don't think I'm the best person to decide how this goes. I think the team would help us out here.